datalad/tests/test_dataset.py

```diff
@@ -0,0 +1,34 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Tests to test implementation of class Dataset
+
+Note: There's not a lot to test by now.
+
+"""
+__author__ = 'Benjamin Poldrack'
+
+import os.path
+from shutil import rmtree
+
+from nose.tools import assert_raises, assert_is_instance, assert_true
+from git.exc import GitCommandError
+
+from datalad.support.dataset import Dataset
+from datalad.tests.utils import with_tempfile
+
+
+@with_tempfile()
+def test_Dataset(pathToTestRepo):
+
+    ds = Dataset(pathToTestRepo, 'http://psydata.ovgu.de/forrest_gump/.git')
```

Comment:
Please do --- it's ruining the stats...
On Feb 24, 2015 3:49 PM, "Yaroslav Halchenko" <notifications@github.com>
wrote:

> In datalad/tests/test_dataset.py
> <https://github.com/datalad/datalad/pull/42#discussion_r25256957>:
>
> > +__author__ = 'Benjamin Poldrack'
> > +
> > +import os.path
> > +from shutil import rmtree
> > +
> > +from nose.tools import assert_raises, assert_is_instance, assert_true
> > +from git.exc import GitCommandError
> > +
> > +from datalad.support.dataset import Dataset
> > +from datalad.tests.utils import with_tempfile
> > +
> > +
> > +@with_tempfile()
> > +def test_Dataset(pathToTestRepo):
> > +
> > +    ds = Dataset(pathToTestRepo, 'http://psydata.ovgu.de/forrest_gump/.git')
>
> now master has that @with_testrepos ;-) so merge and tune away from bulky
> remote datasets
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datalad/datalad/pull/42/files#r25256957>.
>